### ISSUE TITLE
Nan loss is coming after 1 epoch

### ISSUE THREAD

**Issue by JH2-LEE** [noahzn/Lite-Mono#98]
Hello, thanks for great work! I ran the train.py to reproduce the lite-mono, but nan loss is coming after some time.

I just clone the repo and added pretrained file, eigen_zhou split and gt_depth.

This is my comand line.
```
python train.py --data_path ~/data/kitti_data --model_name mytrain --mypretrain ckpts/pretrain/lite-mono-pretrain.pth --num_epochs 30 --png --batch_size 12 --lr 0.0001 5e-6 31 0.0001 1e-5 31
```

I got result as below.
```
Training model named:
   mytrain
Models and tensorboard events files are saved to:
   ./tmp
Training is using:
   cuda
/home/jihui/miniconda3/envs/robodepth/lib/python3.10/site-packages/torchvision/transforms/transforms.py:329: UserWarning: Argument 'interpolation' of type int is deprecated since 0.13 and will be removed in 0.15. Please use InterpolationMode enum.
  warnings.warn(
Using split:
   eigen_zhou
There are 39810 training items and 4424 validation items

Training
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch      0 | examples/s:   1.6 | loss: 0.15450 | time elapsed: 00h00m26s | time left: 00h00m00s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch    250 | examples/s:  22.1 | loss: 0.13400 | time elapsed: 00h03m34s | time left: 23h42m04s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch    500 | examples/s:  16.9 | loss: 0.12268 | time elapsed: 00h06m47s | time left: 22h23m44s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch    750 | examples/s:  17.6 | loss: 0.11716 | time elapsed: 00h09m58s | time left: 21h54m05s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   1000 | examples/s:  18.0 | loss: 0.11960 | time elapsed: 00h13m04s | time left: 21h27m58s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   1250 | examples/s:  19.1 | loss: 0.11480 | time elapsed: 00h16m17s | time left: 21h20m11s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   1500 | examples/s:  19.3 | loss: 0.13234 | time elapsed: 00h19m33s | time left: 21h18m25s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   1750 | examples/s:  16.3 | loss: 0.14224 | time elapsed: 00h22m57s | time left: 21h22m22s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   2000 | examples/s:  16.3 | loss: 0.08798 | time elapsed: 00h26m18s | time left: 21h22m51s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   2250 | examples/s:  20.4 | loss: 0.10477 | time elapsed: 00h29m44s | time left: 21h25m20s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   2500 | examples/s:  18.5 | loss: 0.11061 | time elapsed: 00h33m04s | time left: 21h23m38s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   2750 | examples/s:   9.7 | loss: 0.09842 | time elapsed: 00h36m31s | time left: 21h24m55s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   3000 | examples/s:  19.7 | loss: 0.09168 | time elapsed: 00h39m55s | time left: 21h24m34s
epoch   0 | lr 0.000100 |lr_p 0.000100 | batch   3250 | examples/s:  20.8 | loss: 0.09171 | time elapsed: 00h43m26s | time left: 21h26m52s
Training
epoch   1 | lr 0.000099 |lr_p 0.000099 | batch      0 | examples/s:  12.5 | loss: 0.10012 | time elapsed: 00h44m31s | time left: 21h31m16s
epoch   1 | lr 0.000099 |lr_p 0.000099 | batch    250 | examples/s:  16.4 | loss: 0.11529 | time elapsed: 00h48m03s | time left: 21h32m34s
epoch   1 | lr 0.000099 |lr_p 0.000099 | batch    500 | examples/s:  18.1 | loss: 0.09899 | time elapsed: 00h51m28s | time left: 21h30m36s
epoch   1 | lr 0.000099 |lr_p 0.000099 | batch    683 | examples/s:  19.0 | loss: 0.09388 | time elapsed: 00h54m01s | time left: 21h29m48s
epoch   1 | lr 0.000099 |lr_p 0.000099 | batch    750 | examples/s:  22.0 | loss: 0.10149 | time elapsed: 00h54m59s | time left: 21h30m27s
epoch   1 | lr 0.000099 |lr_p 0.000099 | batch   1000 | examples/s:  21.7 | loss: 0.09889 | time elapsed: 00h58m06s | time left: 21h21m22s
epoch   1 | lr 0.000099 |lr_p 0.000099 | batch   1250 | examples/s:  13.7 | loss: 0.09906 | time elapsed: 01h01m17s | time left: 21h14m09s
epoch   1 | lr 0.000099 |lr_p 0.000099 | batch   1500 | examples/s:  19.6 | loss: nan | time elapsed: 01h04m23s | time left: 21h05m40s
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
NaN or Inf found in input tensor.
/home/jihui/miniconda3/envs/robodepth/lib/python3.10/site-packages/tensorboardX/summary.py:285: RuntimeWarning: invalid value encountered in cast
  tensor = (tensor * 255.0).astype(np.uint8)
NaN or Inf found in input tensor.
```
This is my result.
![Screenshot from 2023-11-29 17-22-18](https://github.com/noahzn/Lite-Mono/assets/62015504/a731b602-f919-4067-b086-4f8bf528630e)
Could you please tell me what’s wrong? Thank you in advance.
